docs/intro/import-guide.rst

```diff
@@ -5,7 +5,9 @@ Importing Your Documentation
    :description lang=en: Import your existing technical documentation from version control into Read the Docs.
 
 
-To import a documentation repository, visit your `Read the Docs dashboard`_ and click Import_.
+To import a public documentation repository, visit your `Read the Docs dashboard`_ and click Import_. For private
+repositories, use the Read the Docs  `commercial solution <https://docs.readthedocs.io/page/commercial/>`_.
```

Comment:
This link should probably be a `ref` or `doc` sphinx role instead of the hardcoded link.

Reply:
Would you be able to make your comment as a suggestion so I can pull it in? I am unfamiliar with sphinx.

Reply:
```suggestion
repositories, use the Read the Docs  :doc:`commercial solution <../commercial/index>`.
```

Thanks for the contribution and I think this is what @humitos means.

Boy, that "commercial" doc needs work. I'll take that on.